static/app/views/insights/sessions/views/overview.tsx

```diff
@@ -36,9 +39,15 @@ export function SessionsOverview() {
                 />
               </ToolRibbon>
             </ModuleLayout.Full>
-            <ModuleLayout.Third>
-              <ErrorFreeSessionsChart />
-            </ModuleLayout.Third>
+            {view === MOBILE_LANDING_SUB_PATH ? (
+              <ModuleLayout.Half>
+                <ErrorFreeSessionsChart groupByRelease />
+              </ModuleLayout.Half>
+            ) : (
+              <ModuleLayout.Third>
+                <ErrorFreeSessionsChart />
+              </ModuleLayout.Third>
+            )}
```

Comment:
This is completely fine, but if you're going to have a lot of differences between Mobile and Web in the Releases UI, you might as well add separate modules for Mobile Releases vs. Web Releases. Up to you, of course, I'm sure you'll notice when it becomes annoying!